crest/Assets/Crest/Crest/Editor/ValidatedInspectorEditor.cs

```diff
@@ -0,0 +1,77 @@
+﻿
+namespace Crest
+{
+    using System;
+    using UnityEditor;
+    using UnityEngine;
+
+    public abstract class ValidatedInspectorEditor : Editor
+    {
+        public override void OnInspectorGUI()
+        {
+            IValidated target = (IValidated)this.target;
+
+            // Enable rich text in help boxes. It is strange that this is how it has to be done. Not sure if 2018.4 bug
+            // or not.
+            GUIStyle richStyle = GUI.skin.GetStyle("HelpBox");
+            richStyle.richText = true;
+
+            var messageTypes = Enum.GetValues(typeof(MessageType));
```

Comment:
Unfortunately this generates garbage iirc. Might be worth just caching this as a static member variable.